gotype.go

```diff
@@ -219,6 +221,10 @@ func (method *Method) BaseResultTypeString(pkg string) string {
 	}
 }
 
+func (method Method) ResourceName() string {
+	return method.hostPackage + "-" + strings.ToLower(method.Name)
```

Comment:
Ah, this answers another question (https://github.com/opentable/go-singularity/pull/7#discussion_r155482670)